v2/internal/platform/win32/structs.go

```diff
@@ -0,0 +1,54 @@
+package win32
+
+type NOTIFYICONDATA struct {
```

Comment:
IIRC this is a duplicate definition with a struct in another package, should we unify this?